### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mocsharp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,16 @@
 💡 If you want to know more about MONAI Deploy WG vision, overall structure, and guidelines, please read [MONAI Deploy](https://github.com/Project-MONAI/monai-deploy) first.
 
 # MONAI Deploy Storage
-Storage layer for MONAI Deploy clinical data pipelines system
+
+The MONAI Deploy Storage library for MONAI Deploy clinical data pipelines system enables users to extend the system to external storage services by implementing the [IStorageService API](src/Storage/API/IStorageService.cs).  The API allows users to plug in any other storage services, such as [AWS S3](https://aws.amazon.com/pm/serv-s3/) and [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/).
```
Comment (dbericat) [inline README.md:9]: Check double space '.  The API'

**Review by dbericat — CHANGES_REQUESTED** [review]
Looks good besides a couple comments.

File: README.md [hunk 2]
```diff
@@ -17,13 +26,13 @@ To participate, please join the MONAI Deploy Workflow Manager weekly meetings on
 
 Join the conversation on Twitter [@ProjectMONAI](https://twitter.com/ProjectMONAI) or join our [Slack channel](https://forms.gle/QTxJq3hFictp31UM9).
 
-Ask and answer questions over on [MONAI Deploy Workflow Manager's GitHub Discussions tab](https://github.com/Project-MONAI/monai-deploy-workflow-manager/discussions).
+Ask and answer questions over on [MONAI Deploy's GitHub Discussions tab](https://github.com/Project-MONAI/monai-deploy/discussions).
```
[the remark this answers is not in the document]
Comment (mocsharp) [reply README.md]: Updated to https://github.com/Project-MONAI/monai-deploy-storage/discussions

File: README.md (hunk 1 above)
Comment (mocsharp) [reply README.md]: Fixed.

**Pull request merged**